Applications/Custom/mae_loss.h

```diff
@@ -43,44 +43,37 @@ class MaeLossLayer final : public nntrainer::LayerV1 {
   using nntrainer::LayerV1::setProperty;
 
   /**
-   * @brief     set Property of layer, currently only "exponent is accepted"
+   * @brief     set Property of layer
```

Comment:
update todo on line 26 to update to LayerV2.

Reply:
Done